### PR TITLE
update libxmtp for epoch and members fixes

### PR DESCRIPTION
**Pull request by cameronvoell**

### CODE REVIEW

**Review by nplasterer — APPROVED** [review]

**Review by alexrisch — APPROVED** [review]

File: library/src/main/java/xmtpv3.kt [hunk 1]
```diff
@@ -376,11 +380,11 @@ private fun findLibraryName(componentName: String): String {
     if (libOverride != null) {
         return libOverride
     }
-    return "uniffi_xmtpv3"
+    return "xmtpv3"
```
Comment (nplasterer) [inline library/src/main/java/xmtpv3.kt:383]: Seems like with the new uniffi version we use in rust this keeps changing this to remove the uniffi and that breaks everything inside of android. I think I'd like to look more into why this breaks android. But Just going to undo this change for now to get a release out.
